code/modules/GOAP/cyka_blyat/actions.dm

```diff
@@ -0,0 +1,339 @@
+/datum/goap_action/russian
+	var/saytext = "Cyka" // Russians can speak so why not!
+
+/datum/goap_action/russian/Perform(mob/living/simple_animal/hostile/russian/A)
+	A.say(saytext)
+	return ..()
+	
+/datum/goap_action/russian/attack
+	name = "Attack"
+	cost = 5 // reload + firing = 3, new mag + reload + firing = 4, so this comes if you can't fire/reload+fire/grab mag+reload+fire
+	saytext = "Moving in to melee!"
+
+/datum/goap_action/russian/attack/New()
+	..()
+	preconditions = list()
+	preconditions["enemyDead"] = FALSE
+	effects = list()
+	effects["enemyDead"] = TRUE
+
+/datum/goap_action/russian/attack/AdvancedPreconditions(atom/agent, list/worldstate)
+	var/list/viewl = view(10, agent)
+	for(var/mob/living/carbon/C in viewl)
+		if(C && !C.stat)
+			target = C
+	return (target != null)
+
+/datum/goap_action/russian/attack/RequiresInRange()
+	return TRUE
+
+/datum/goap_action/russian/attack/Perform(mob/living/simple_animal/hostile/A)
+	var/mob/living/carbon/C = target
+	C.attack_animal(A)
+	action_done = TRUE
+	return ..()
+
+/datum/goap_action/russian/attack/CheckDone(atom/agent)
+	return action_done
+
+
+/datum/goap_action/russian/attack_ranged
+	name = "Attack Ranged"
+	cost = 2
+	saytext = "Suppressive fire!"
```

Comment:
change these back to the original lines, we have fixed Cyrillic support now iirc

Reply:
The original lines are still there, this one didn't have a line so I added it